Makefile

```diff
@@ -48,21 +48,40 @@ NEWEST_GO_FILE = $(shell find $(SRC_DIRS) -name \*.go -exec $(STAT) {} \; \
 TYPES_FILES    = $(shell find pkg/apis -name types.go)
 GO_VERSION     = 1.8
 
+ALL_ARCH=amd64 arm arm64 ppc64le s390x
```

Comment:
The list here doesn't seem to match what I see in: https://github.com/kubernetes/kubernetes/blob/master/build/build-image/cross/Dockerfile#L21   in particular the "armhf" - does it matter?